### ISSUE TITLE
Lots of invalid value of suppress popups

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#5801]
```
ERROR [2018-08-30 12:42:49Z]: Error while loading global rule set.
Microsoft.CodeAnalysis.InvalidRuleSetException: The attribute Action has an invalid value of Suppress.
  at Microsoft.CodeAnalysis.RuleSetProcessor.ReadAction (System.Xml.Linq.XElement node, System.Boolean allowDefault) [0x00068] in /_/src/Compilers/Core/Portable/RuleSet/RuleSetProcessor.cs:237 
  at Microsoft.CodeAnalysis.RuleSetProcessor.ReadRule (System.Xml.Linq.XElement ruleNode, System.String analyzer, System.String space) [0x0000b] in /_/src/Compilers/Core/Portable/RuleSet/RuleSetProcessor.cs:184 
  at Microsoft.CodeAnalysis.RuleSetProcessor.ReadRules (System.Xml.Linq.XElement rulesNode) [0x0004c] in /_/src/Compilers/Core/Portable/RuleSet/RuleSetProcessor.cs:162 
  at Microsoft.CodeAnalysis.RuleSetProcessor.ReadRuleSet (System.Xml.Linq.XElement ruleSetNode, System.String filePath) [0x00058] in /_/src/Compilers/Core/Portable/RuleSet/RuleSetProcessor.cs:112 
  at Microsoft.CodeAnalysis.RuleSetProcessor.LoadFromFile (System.String filePath) [0x0006b] in /_/src/Compilers/Core/Portable/RuleSet/RuleSetProcessor.cs:89 
  at Microsoft.CodeAnalysis.RuleSet.LoadEffectiveRuleSetFromFile (System.String filePath) [0x00000] in /_/src/Compilers/Core/Portable/RuleSet/RuleSet.cs:253 
  at MonoDevelop.Ide.TypeSystem.MonoDevelopRuleSetManager.GetGloabalRuleSet () [0x00025] in /Users/vsts/agent/2.139.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.TypeSystem/MonoDevelopRuleSetManager.cs:118 
```

This stacktrace is coupled with a lot of popups. Happens while loading/building https://github.com/Microsoft/vs4mac-tfvc

> VS bug [#674233](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/674233)

**Comment by mrward** [comment]
This is happening for me with a local build of the latest master for each project in any solution that is opened. Each project shows a dialog about an 'error while loading global rule set'. Should we be displaying a dialog here? If so then can we limit it so it does not get displayed for each project? This is with Mono 5.12.0.301.

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5807 merged by: Mike Kr??ger  
 Author: mono  
 Commit: 8cd3a99c9eb75ae9d2ffc27e7cb6fdf00c1babfd (xamarin/monodevelop)
